build.sbt

```diff
@@ -825,12 +824,11 @@ lazy val sharedReleaseProcess = Seq(
   releaseProcess := Seq[ReleaseStep](
     checkSnapshotDependencies,
     inquireVersions,
-    runClean,
-    releaseStepCommand("validate"),
+    releaseStepCommandAndRemaining("+validate"),
```

Comment:
Is this a switch from running `validate` from the current version to all versions? If so, does this lead to OOMs or anything?

Reply:
Not really. Before the validate is still run against all scala versions because the `crossScalaVersions` settings on the root module and the `releaseCrossBuild := true`. sbt release automatically runs certain steps on each modules against all crossScalaVersions set on the root module. Now, with the new sbt version , the `+validate` will performe on each aggregated module correctly on their own `crossScalaVersions`. The root module's `crossScalaVersions` should set as `Nil`, otherwise it will conflict with the aggregrated module's `crossScalaVersion`.